cobc/config.def

```diff
@@ -385,3 +404,14 @@ CB_CONFIG_SUPPORT (cb_vsam_status, "vsam-status",
 CB_CONFIG_SUPPORT (cb_self_call_recursive, "self-call-recursive",
 	_("CALL to own PROGRAM-ID implies RECURSIVE attribute"))
 
+CB_CONFIG_SUPPORT (cb_record_contains_depending_clause, "record-contains-depending-clause",
+	_("DEPENDING clause in RECORD CONTAINS"))
+
+CB_CONFIG_BOOLEAN (cb_relax_paragraph_name_uniqueness, "relax-paragraph-name-uniqueness",
```

Comment:
This seems to implement https://sourceforge.net/p/gnucobol/feature-requests/260/ - doesn't it? In this case you'd likely reference that in the changelog entry. In any case that issue may have some relevant data to consider in.